frontend/src/components/ui/LoadMoreButton.tsx

```diff
@@ -6,19 +6,21 @@ export default function LoadMoreButton({
   isLoading,
   setSize,
   size,
+  entity,
 }: {
   results: any[];
   totalResults: number;
   isLoading: boolean;
   setSize: React.Dispatch<React.SetStateAction<number>>;
   size: number;
+  entity: string;
 }) {
   return (
     <button
       className="bg-secondary text-white opacity-90 hover:opacity-100 rounded-lg py-2 px-6 mt-4 disabled:opacity-50 disabled:cursor-not-allowed"
       onClick={() => setSize(size + 1)}
       disabled={isLoading || results.length >= totalResults}>
-      {results.length >= totalResults ? "No more characters" : "Load more"}
+      {results.length >= totalResults ? `No more ${entity}` : "Load more"}
     </button>
   );
 }
```

Comment:
same with that change, it's a good refactoring idea, but please open a new pr after this one has been merged.

Reply:
yes for sure. thank you for the helpful feedback.